test/testdrive/mzcompose.py

```diff
@@ -25,7 +25,11 @@
     SchemaRegistry(),
     Redpanda(),
     Localstack(),
-    Materialized(),
+    Materialized(
+        additional_system_parameter_defaults={
+            "persist_fast_path_limit": "1000",
```

Comment:
This does not belong here. It is best to put it in `misc/python/materialize/mzcompose/__init__.py` , `DEFAULT_SYSTEM_PARAMETERS` , so that it applies to all mzcompose-based tests, including e.g. Zippy and the various random query generators. The default should ideally be the same as the one used in production.

Reply:
I wrote a bit about the rationale for this in the PR description.

At the moment I would prefer to leave it disabled entirely in CI than to have it enabled everywhere, because we have not yet done the hardening or testing that's required to be comfortable turning this on in prod. Let me know if you'd like that better, or if we should find some other way to get a small amount of coverage in the meantime.

Reply:
Historically we have had a lot of trouble keeping the settings used in the CI aligned with production defaults and/or picking the best settings to use for testing regardless of the production defaults. Hence the need to keep them all unified in `DEFAULT_SYSTEM_PARAMETERS` . So putting a custom default in testdrive/mzcompose.py would be a step backwards in that sense.

Consider either enabling this for the entire CI or augmenting your existing SLT file with a few SELECTs.